distutils_precedence.pth

```diff
@@ -0,0 +1 @@
+import os; (os.environ.get('SETUPTOOLS_USE_DISTUTILS', 'stdlib') == 'local' and __import__('_distutils_importer'))
```

Comment:
What do you think about unconditionally importing the functionality and moving the conditional there? 
```suggestion
_distutils_importer
```